doc/Setup.md

```diff
@@ -35,7 +63,7 @@ You'll have to point to the correct cfitsio and fftw3 libraries and headers for
 $> make
 ```
 
-3) Test `PhoSim`.
+4) Test `PhoSim`.
```

Comment:
Perhaps we can omit the sections below on `Gravitational Lens Sprinkling Setup` and `Supernova Sprinkling Setup`?  They are effectively covered in `Sims_phoSimGen.md`